design-docs/CLI-Subscriptions.md

```diff
@@ -222,13 +222,13 @@ Generate a subscriptions request source JSON for a catalog.
 Options:
```

Comment:
1. The example above, line 210, does not work. It is missing a `--name` flag. 
2. `--clip` is no longer an option, looks like it got replaced by `--tools`.

A working example would look something like this 
```
planet subscription request --name "some name" --source source.json --tools geom.json --delivery delivery.json | planet subscriptions create -
```